test/integration/util/util.go

```diff
@@ -65,17 +66,16 @@ func (m *MinikubeRunner) EnsureRunning() {
 }
 
 func (m *MinikubeRunner) SetEnvFromEnvCmdOutput(dockerEnvVars string) error {
-	lines := strings.Split(dockerEnvVars, "\n")
-	var envKey, envVal string
+	re := regexp.MustCompile("export (.+?)=(.+?)\n")
```

Comment:
This will only be the case for certain shells (Linux/osx). Guess that's OK for integration tests?